R/zzz.R

```diff
@@ -282,3 +282,4 @@ has_missing_count <- FALSE
 kept_levels <-expr(TRUE)
 levels_to_keep <- NULL
 break_ties <- NULL
+stats_as_columns <- NULL
```

Comment:
```suggestion
stats_as_columns <- FALSE
```

That should make some of the code more straight-forward when you do `if (stats_as_columns)`

Reply:
Updated